scripts/resources.sh

```diff
@@ -1,34 +1,42 @@
 #!/usr/bin/env nix-shell
-#! nix-shell -i bash -p bash jq
+#!nix-shell -i bash -p bash jq
 # shellcheck shell=bash
 
-##
-# This file reads the resource IDs from one deployment and feeds them to
-# another deployment via `nixops set-args`.
 #
-# Currently, we pull these values from the `deployer` network:
+# Usage: ./resources.sh <src_deployment> <tgt_deployment>
+#
+# This script reads the resource IDs from one deployment and feeds them
+# to another deployment via `nixops set-args`.
+#
+# We pull these values from the `deployer` network:
+#
 # - VPC ID
 # - VPC CIDR block
 # - Route Table ID
 #
 # These resources are necessary to link an independent deployment into the same
 # network.
-#
-# E.g. to copy resources from the "deployer" to the "staging" cluster:
-# $ ./resources.sh deployer staging
 
 set -euo pipefail
 [[ -n ${DEBUG:-} ]] && set -x
 
-from=${1:-staging}
-to=${2:-disciplina}
+################################################################################
+################################################################################
+
+
+from="$1"
+to="$2"
+
+[ -n "$from" -o -n "$to" ] || {
+  echo >&2 "Usage: $0 <src_deployment> <tgt_deployment>"
```

Comment:
No need to abbreviate source and target here and above. Keep things readable unless it *needs* to be short.